src/messages/javascript.js

```diff
@@ -152,7 +154,7 @@ function temporaryAPI(api) {
     code: apiToMessage(api),
     message: i18n._(`"${api}" can cause issues when loaded temporarily`),
     description: i18n._(oneLine`This API can cause issues when loaded
-      temporarily using about:debugging in Firefox unless you specify
+      temporarily using about:debugging in Thunderbird unless you specify
```

Comment:
`applications` is deprecated and we should use `browser_specific_settings` only. Maybe remove it here, so we do not have to deal with it later anymore?